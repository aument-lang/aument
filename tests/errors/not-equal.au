struct Class { a, b }

my_struct != new Class;