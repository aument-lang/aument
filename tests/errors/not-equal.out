parser error(5) in -: unknown variable 'my_struct'
5 | my_struct != new Class;
    ^^^^^^^^^
